From b136ebc3e53ce3ff523c1995fb6b8ca5decc8545 Mon Sep 17 00:00:00 2001
From: Jiri Pirko <jpirko@redhat.com>
Date: Mon, 15 Sep 2014 09:32:04 -0400
Subject: [net] xfrm4: Remove duplicate semicolon

Message-id: <1410773525-8735-2-git-send-email-jpirko@redhat.com>
Patchwork-id: 94792
O-Subject: [patch rhel7 net 20/21] xfrm4: Remove duplicate semicolon
Bugzilla: 1091561
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

BZ1091561
https://brewweb.devel.redhat.com/taskinfo?taskID=7954687

upstream commit 1759389e8af46d724220785bf710b7bdbebdfa48
Author: Christoph Paasch <christoph.paasch@uclouvain.be>
Date:   Sat Jun 28 14:12:44 2014 +0200

    xfrm4: Remove duplicate semicolon

    3328715e6c1fc (xfrm4: Add IPsec protocol multiplexer) adds a
    duplicate semicolon after the return-statement.

    Although it has no negative impact, the second semicolon should be
    removed.

Signed-off-by: Jiri Pirko <jpirko@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/ipv4/xfrm4_protocol.c b/net/ipv4/xfrm4_protocol.c
index b52c390..42e26e1 100644
--- a/net/ipv4/xfrm4_protocol.c
+++ b/net/ipv4/xfrm4_protocol.c
@@ -124,7 +124,7 @@ static int xfrm4_ah_rcv(struct sk_buff *skb)
 
  for_each_protocol_rcu(ah4_handlers, handler)
   if ((ret = handler->handler(skb)) != -EINVAL)
-   return ret;;
+   return ret;
 
  icmp_send(skb, ICMP_DEST_UNREACH, ICMP_PORT_UNREACH, 0);
 
-- 
1.7.1